flink-examples/src/main/java/io/pravega/examples/flink/alert/HighCountAlerter.java

```diff
@@ -54,22 +56,24 @@ public static void main(String[] args) throws Exception {
         // initialize the parameter utility tool in order to retrieve input parameters
         ParameterTool params = ParameterTool.fromArgs(args);
 
-        // create Pravega helper utility for Flink using the input paramaters
-        FlinkPravegaParams helper = new FlinkPravegaParams(params);
+        PravegaConfig pravegaConfig = PravegaConfig
+                .fromParams(params)
+                .withDefaultScope(Constants.DEFAULT_SCOPE);
 
-        // get the Pravega stream from the input parameters
-        StreamId streamId = helper.getStreamFromParam(Constants.STREAM_PARAM,
-                                                      Constants.DEFAULT_STREAM);
+        // create the Pravega input stream (if necessary)
+        Stream stream = Utils.createStream(
+                pravegaConfig,
+                params.get(Constants.STREAM_PARAM, Constants.DEFAULT_STREAM));
 
-        // create the Pravega stream is not exists.
-        helper.createStream(streamId);
-
-        // initialize Flink execution environment
+        // initialize the Flink execution environment
         final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 
-        // create the Pravega stream reader
-        long startTime = 0;
-        FlinkPravegaReader<String> reader = helper.newReader(streamId, startTime, String.class);
+        // create the Pravega source to read a stream of text
+        FlinkPravegaReader<String> reader = FlinkPravegaReader.<String>builder()
+                .withPravegaConfig(pravegaConfig)
```

Comment:
while testing in secure Pravega cluster, I have noticed that we also need to configure the reader group scope name as well if it the stream's scope is different from the default scope

Reply:
The scope is set in PravegaConfig. Is that not sufficient?